contracts/test/integration/L2PlusSubnet.t.sol

```diff
@@ -219,25 +219,6 @@ contract L2PlusSubnetTest is Test, IntegrationTestBase {
         sendCrossMessageFromChildToParentWithResult(params);
     }
 
-    function testL2PlusSubnet_TokenMixed_SendCrossMessageFromChildToParentWithOkResult() public {
```

Comment:
these tests are removed because tokenL2 to token L3 xnet value transfer does not work.